### ISSUE TITLE
Selecting no text preprocessing raises an error

### ISSUE THREAD

**Issue by wiitt** [EveryVoiceTTS/EveryVoice#523]
The wizard raises the following error when you select no text preprocessing and it tries to iterate trough the response.
```╭─────────────────────────────── Traceback (most recent call last) ────────────────────────────────╮
│ /gpfs/fs3c/nrc/dt/vlg000/repositories/EveryVoice/everyvoice/cli.py:131 in new_project            │
│                                                                                                  │
│   128 def new_project():                                                                         │
│   129 │   from everyvoice.wizard.main_tour import get_main_wizard_tour                           │
│   130 │                                                                                          │
│ ❱ 131 │   get_main_wizard_tour().run()                                                           │
│   132                                                                                            │
│   133                                                                                            │
│   134 # Add preprocess to root                                                                   │
│                                                                                                  │
│ /gpfs/fs3c/nrc/dt/vlg000/repositories/EveryVoice/everyvoice/wizard/__init__.py:171 in run        │
│                                                                                                  │
│   168 │                                                                                          │
│   169 │   def run(self):                                                                         │
│   170 │   │   for _, _, node in RenderTree(self.root):                                           │
│ ❱ 171 │   │   │   node.run()                                                                     │
│   172 │                                                                                          │
│   173 │   def visualize(self):                                                                   │
│   174 │   │   for pre, _, node in RenderTree(self.root):                                         │
│                                                                                                  │
│ /gpfs/fs3c/nrc/dt/vlg000/repositories/EveryVoice/everyvoice/wizard/__init__.py:111 in run        │
│                                                                                                  │
│   108 │   │   │   │   │   self.state[self.name] = self.response                                  │
│   109 │   │   │   │   return self.response                                                       │
│   110 │   │   │   finally:                                                                       │
│ ❱ 111 │   │   │   │   self.effect()                                                              │
│   112 │   │   else:                                                                              │
│   113 │   │   │   self._validation_failures += 1                                                 │
│   114 │   │   │   if self._validation_failures > 20:                                             │
│                                                                                                  │
│ /gpfs/fs3c/nrc/dt/vlg000/repositories/EveryVoice/everyvoice/wizard/dataset.py:766 in effect      │
│                                                                                                  │
│   763 │   │   │   │   │   │   )                                                                  │
│   764 │   │   │   │   │   )                                                                      │
│   765 │   │   │   # Process any dataset-specified cleaners                                       │
│ ❱ 766 │   │   │   for process in self.response:                                                  │
│   767 │   │   │   │   process_fn = self.process_lookup[process]["fn"]                            │
│   768 │   │   │   │   for item in tqdm(                                                          │
│   769 │   │   │   │   │   self.state["filelist_data"],                                           │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
TypeError: 'NoneType' object is not iterable```

**Comment by roedoejet** [comment]
I seem to remember this has been fixed...we need to confirm though.

**Comment by joanise** [comment]
Fixed by a80e7e79d7
